code/modules/projectiles/ammo_datums.dm

```diff
@@ -1423,6 +1444,42 @@ datum/ammo/bullet/revolver/tp44
 /datum/ammo/rocket/ltb/drop_nade(turf/T)
 	explosion(T, 0, 4, 6, 7)
 
+/datum/ammo/rocket/heavyrr
+	name = "75mm round"
+	icon_state = "heavyrr"
+	hud_state = "shell_he"
+	hud_state_empty = "shell_empty"
+	flags_ammo_behavior = AMMO_EXPLOSIVE|AMMO_ROCKET|AMMO_SUNDERING
+	accuracy = 40
+	accurate_range = 15
+	max_range = 40
+	penetration = 200
+	damage = 200
+	sundering = 30
+	handful_amount = 1
+
+/datum/ammo/rocket/heavyrr/drop_nade(turf/T)
+	explosion(T, 0, 4, 6, 5)
```

Comment:
Do I even want to know why this adds an aimmode sadar with inbuilt scope that you can grab for *dropship fab* points?

Reply:
> because you can't move it and can only fire in one direction

You do know you can turn deployables when manning them yes?